packages/core/src/registry/registry.ts

```diff
@@ -0,0 +1,69 @@
+import { RouteHandlerMethod } from "fastify";
+import { RESTMethods } from "../loader/rest";
+
+export type RegistryMock = {
+  method: RESTMethods;
+  path: string;
+  resolver: RouteHandlerMethod;
+  isDefault: boolean;
+  name: string;
+};
+
+export default class Registry {
```

Comment:
Then we should delete old registry please

Reply:
Done 👍